File: servicestack/src/JsonService.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using ServiceStack.ServiceHost;
using ServiceStack.ServiceInterface;

namespace ServiceStackBenchmark
{
    [Route("/json")]
    public class Json
    {
        public string message { get; set; }
    }

    public class JsonService : Service
    {
        public Json Get(Json empty)
        {
            return new Json { message = "Hello, World!" };
        }
    }
}

File: frameworks/Java/baratine/src/main/java/testTechempowerBaratine/JsonService.java
package testTechempowerBaratine;

import io.baratine.service.Result;
import io.baratine.service.Service;
import io.baratine.web.http.Get;

@Service
public class JsonService
{
  @Get("/json")
  public void hello(Result<HelloWorld> result)
  {
    result.ok(new HelloWorld("Hello, World!"));
  }

  public static class HelloWorld {
    private String message;

    public HelloWorld(String msg)
    {
      this.message = msg;
    }
  }
}
